tests/test_low_pass_filter.py

```diff
@@ -0,0 +1,13 @@
+# This script tests the low_pass_filter function
+
+import numpy as np
+import pytest
+from AudioFilters.low_pass_filter import low_pass_filter
+
+input_signal = np.genfromtxt('data/bark.csv', dtype = 'float32')
+
+def test_low_pass_filter_attenuates_frequencies_below_1000_cutoff():
```

Comment:
I think the name of the test should now say `above_1000_cutoff` instead of `below_1000_cutoff`, since the filter is now low pass.